nemo/collections/nlp/models/language_modeling/megatron_gpt_adapter_model.py

```diff
@@ -51,6 +51,11 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
         assert (
```

Comment:
Not sure if important, but should we also add a validation step to check if all necessary keys are present in the given config (like adapter_dim or dropout)?

Reply:
dropout is not required, but I can add one for adapter_dim.